### DISCUSSION TITLE
Help please. Trying to install and getting following error

### DISCUSSION THREAD

**Discussion by Starhawk88** [oobabooga/text-generation-webui#5872] [Q&A]
python: can't open file 'C:\\Users\\hatha\\AppData\\Local\\Temp\\cbbbf079-a02f-4bbd-9182-05efbcc3d5e9_text-generation-webui-main.zip.5e9\\text-generation-webui-main\\one_click.py': [Errno 2] No such file or directory

**Comment by natbutter** [comment]
Unzip the text-generation-webui-main.zip folder first before trying to install it.
